bgpd/bgp_nexthop.c

```diff
@@ -919,7 +919,7 @@ static int show_ip_bgp_nexthop_table(struct vty *vty, const char *name,
 		}
 		tree = import_table ? &bgp->import_check_table
 				    : &bgp->nexthop_cache_table;
-		bnc = bnc_find(tree[family2afi(nhop.family)], &nhop, 0, 0);
+		bnc = bnc_find(&(*tree)[family2afi(nhop.family)], &nhop, 0, 0);
 		if (!bnc) {
```

Comment:
I manually tested the previous pull request (https://github.com/FRRouting/frr/pull/12038), and it was ok.
this is not a pointer issue.